packages/flutter_tools/lib/src/android/android_studio.dart

```diff
@@ -77,7 +80,10 @@ class AndroidStudio implements Comparable<AndroidStudio> {
       // ignored, installPath will be null, which is handled below
     }
     if (installPath != null && fs.isDirectorySync(installPath)) {
-      return new AndroidStudio(installPath, version: version);
+      return new AndroidStudio(
+          installPath,
+          version: version,
+          studioAppName: studioAppName);
```

Comment:
nit: newline before `)`

Reply:
(and a trailing comma on the previous line)

Reply:
Done